src/user-guide/writing-suites.rst

```diff
@@ -1063,7 +1044,7 @@ Multiple integer exclusions are also valid in the same way as the syntax
 in :ref:`excluding-dates`. Integer exclusions may be a list of single
 integer points, an integer sequence, or a combination of both:
 
-.. code-block:: cylc
+.. code-block:: sub
 
    R/P1!(2,3,7)  # Run with step 1 to the final cycle point,
                  # but not at points 2, 3, or 7.
```

Comment:
FYI the language "sub" is something I created for Rose for code examples which aren't actually bash but where you want comments to be highlighted. It also highlights `<substitutions>` (hence the name).